src/module.js

```diff
@@ -117,7 +117,7 @@ _.extend(Marionette.Module.prototype, Backbone.Events, {
       Marionette,
       Backbone.$, _,
       customArgs
-    ]);
+    ], true);
```

Comment:
I can think of only two options:
 1. test for lodash `_.name === 'lodash'`
 2. use `_.noConflict()` and override `flatten`

Second option seems to be more future proof as there might be more inconsistencies in the future.

What are your thoughts?